docs/pages/posts/2022/8/29/secureum-bootcamp-epoch-august-race-9.mdx

```diff
@@ -291,4 +297,20 @@ The use of the `CALLDATACOPY` opcode will copy the full `CALLDATASIZE` to memory
 The slot-hash calculation has already finished when the `assembly` block begins, therefore there should not be any interference by overwriting the scratch space that was used for it.
 </HiddenSolution>
 
----
\ No newline at end of file
+---
+
+import { Card, Cards } from 'nextra/components'
+import { NewsletterIcon } from '@components/icons'
+
+<Cards>
+  <Card
+    icon={<NewsletterIcon />}
+    title="← Secureum Bootcamp Epoch∞ - July RACE #8"
+    href="/posts/2022/7/25/secureum-bootcamp-epoch-july-race-8/"
+  />
+  <Card
+    icon={<NewsletterIcon />}
+    title="Secureum Bootcamp Epoch∞ - October RACE #10 →"
+    href="/posts/2022/10/3/secureum-bootcamp-epoch-october-race-10"
+  />
```

Comment:
Is missing the discussed additional link to A-MAZE.

Same applies to all other places where there's both a RACE next and something else.